src/types.d.ts

```diff
@@ -132,15 +132,15 @@ export interface IServiceEndpointData {
   tenant_id: string;
 }
 
+export interface IVariableGroupDataVariable {
+  [key: string]: AzureKeyVaultVariableValue;
+}
+
 export interface IVariableGroupData {
   name: string;
   description: string;
   type: string;
-  variables: [
-    {
-      [key: string]: AzureKeyVaultVariableValue;
-    }
-  ];
+  variables: [IVariableGroupDataVariable];
```

Comment:
I think this is supposed to be `variables: IVariableGroupDataVariable[]`

Reply:
@sarath-p ^^ is that correct?

Reply:
@evanlouie, you are right. I will change and test again. thanks